src/gino/ext/__init__.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from importlib.abc import MetaPathFinder
+from importlib.util import find_spec as _find_spec
+
+from importlib_metadata import entry_points
+
+
+class DependencyFinder(MetaPathFinder):
```

Comment:
shouldn't be called `DependencyFinder` - plugins are not dependencies.

Reply:
Rename to PluginFinder